core/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "libtock-core"
+version = "0.1.0"
+authors = ["torfmaster <briefe@kebes.de>", "Woyten <woyten.tielesch@online.de>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
Remove the comment pointing to Cargo's docs.

Reply:
Sure.

Reply:
fixed.